action.yml

```diff
@@ -23,11 +23,14 @@ inputs:
   fields-to-check-for-values:
     description: 'A comma separated string of fields that will be checked for values. Spaces will be trimmed.'
     required: true
-    default: 'customfield_16323, customfield_11506' # These correlate to Technical Approval and Stakeholder Approval fields in im-open's Jira
+    default: 'customfield_16323, customfield_11506, customfield_11507' # These correlate to Technical Approval, Stakeholder Approval and QA approval fields in im-open's Jira.
   check-parent-task:
     description: "A flag determining whether the parent task will be checked in the event a value isn't found on the task itself."
     required: false
     default: 'false'
```

Comment:
However the `check-parent-task:` defaults to `false`. I guess we will just have to remember to set this to true for our workflows. Alternatively we could default it to `true` since this is really meant for our project at this point.